storybook/src/admin/Dialog.tsx

```diff
@@ -0,0 +1,30 @@
+import { Dialog } from "@comet/admin";
+import { DialogContent, Typography } from "@mui/material";
+import { storiesOf } from "@storybook/react";
+import React from "react";
+
+const textContent =
+    "Dialog content. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.";
+
+storiesOf("@comet/admin/Dialog", module)
```

Comment:
This should probably be combined with the existing `Dialog` story: https://storybook.comet-dxp.com/?path=/story/comet-admin-mui--dialog

Maybe extend the existing one and add knobs for `title` and `onClose`, similar to `Dialog size`: 

https://github.com/vivid-planet/comet/blob/f8ae0843c9fc3b12f803b0bf961d898eb11e5ac5/storybook/src/admin/mui/Dialog.tsx#L29